torchtitan/config_manager.py

```diff
@@ -194,6 +194,15 @@ def __init__(self):
             action="store_true",
             help="Whether the fused implementation(CUDA only) is used.",
         )
+        self.parser.add_argument(
+            "--optimizer.backward",
```

Comment:
I somehow think we can have a better name for this. `backward` is less informative.
How about `early_step_in_backward` or `async_step`. cc: @fegin

Reply:
Use `early_step_in_backward` right now